acra-core/src/main/java/org/acra/plugins/ServicePluginLoader.java

```diff
@@ -46,17 +46,24 @@
     private <T extends Plugin> List<T> loadInternal(@NonNull Class<T> clazz, Predicate<T> shouldLoadPredicate) {
         List<T> plugins = new ArrayList<>();
         //noinspection ForLoopReplaceableByForEach
-        for (final Iterator<T> iterator = ServiceLoader.load(clazz, getClass().getClassLoader()).iterator(); iterator.hasNext(); ) {
+        final ServiceLoader<T> serviceLoader = ServiceLoader.load(clazz, getClass().getClassLoader());
+        if (ACRA.DEV_LOGGING) ACRA.log.d(LOG_TAG, "ServicePluginLoader loading services from ServiceLoader : " + serviceLoader);
```

Comment:
In which case would this print useful information?

Reply:
All cases.
Knowing which ServiceLoader you have it highly useful.

Remember this log only shows when a developer has switched on ACRA.DEV_LOGGING.